contracts/polygon/connectors/quickswap/helpers.sol

```diff
@@ -0,0 +1,184 @@
+pragma solidity ^0.7.0;
+
+import {TokenInterface} from "../../common/interfaces.sol";
+import {DSMath} from "../../common/math.sol";
+import {Basic} from "../../common/basic.sol";
+import {IQuickSwapRouter, IQuickSwapFactory} from "./interface.sol";
+
+abstract contract Helpers is DSMath, Basic {
+    /**
+     * @dev IQuickSwapRouter
+     */
+    IQuickSwapRouter internal constant router =
+        IQuickSwapRouter(0xd9e1cE17f2641f24aE83637ab66a2cca9C378B9F);
```

Comment:
The quickswap router address on polygon chain is - `0xa5E0829CaCEd8fFDD4De3c43696c57F7D7A678ff`

Reply:
commited